lib/api/tensor-transforms.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+// Exposes a set of light-weight tensor(s) transformations
+
+import {Tensor} from './tensor';
+
+export interface UtilityTensorCreators {
+  zeros(shape: ReadonlyArray<number>, dtype: Tensor.NumberOrBoolType): Tensor;
```

Comment:
Do we need Jsdoc comments for these methods in this file? @fs-eire - what do you think ?